src/hotspot/src/share/vm/runtime/globals.hpp

```diff
@@ -4000,6 +4000,9 @@ class CommandLineFlags {
   product(bool, UseLockedTracing, false,                                    \
           "Use locked-tracing when doing event-based tracing")              \
                                                                             \
+  diagnostic(bool, UseUnalignedAccesses, false,                             \
```

Comment:
this flag is suspicious 
i) `AvoidUnalignedAccesses` above seems to play same role. 
i) hotspot doesn't have code to read this variable.

Reply:
Will delete UseUnalignedAccesses.